src/csharp/Grpc.Core.Tests/ServerTest.cs

```diff
@@ -81,6 +82,21 @@ public void PickUnusedPort()
         }
 
         [Test]
+        public void StartThrowsWithUnboundPorts()
+        {
+            int twiceBoundPort = 9999;
```

Comment:
I don't know this code base's style well, but `twiceBoundPort` can be marked `const`.